pootle/static/js/browser/components/TopContributors.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) Pootle contributors.
+ *
+ * This file is a part of the Pootle project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React from 'react';
+
+import Avatar from 'components/Avatar';
+
+
+function t(string, args) {
```

Comment:
Shouldn't this function be in `shared/utils/i18n.js`, and shouldn't it have a different name since there is another `t` function that does a similar thing?